framework/base/Profiler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\base;
+
+use Yii;
+use yii\log\Logger;
+
+/**
+ * Profiler is the class for application profiling.
+ *
+ * It is a basic implementation of profiling class. It has [[begin()]] and
```

Comment:
This paragraph could be removed since it repeats previous one and describes interface that we already have in the code.